core/pom.xml

```diff
@@ -78,6 +78,12 @@
 			<artifactId>logback-classic</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>uk.org.webcompere</groupId>
+			<artifactId>system-stubs-jupiter</artifactId>
+			<version>${version.junit5.jupiter.system.stubs}</version>
```

Comment:
I meant rather specifying version in [dependencyManagement](https://github.com/xtf-cz/xtf/blob/master/pom.xml#L91), `<version>` is not necessary then. See how other dependencies are defined.

Reply:
you are right, now is fixed